Documentation/components/iptables/pcn-iptables.rst

```diff
@@ -129,4 +143,20 @@ Limitations
 ^^^^^^^^^^^
 
 - It requires your network interfaces to support XDP Native mode
-- If any interface is not supporting XDP, on such interface traffic is not filtered
\ No newline at end of file
+- If any interface is not supporting XDP, on such interface traffic is not filtered
```

Comment:
Should we write this as follows?
pcn-iptables operates only on interfaces that support XDP (native or generic?) mode; traffic is not filtered on interfaces that support only eBPF TC programs.